codigo/Código/src/código/Código.java

```diff
@@ -0,0 +1,254 @@
+                            }
+                        }
+                    }
+                }
+                contador = contador + 1;
+            }else{
+            System.out.println("Opcion incorrecta");
+            }
+            entrada.nextLine();
+            System.out.println("\"Ingrese *1* si quiere seguir en el ciclo o *2*"
+                    + " si quiere salir del ciclo:\\n \"");
+            salida = entrada.nextInt();
+            if(salida == 2){
+                bandera = false;
+            }
+            obtenerMensaje(contador);
+        }
+
+    }
+    public static String crearFacebook(){
+        Scanner entrada = new Scanner(System.in);
+        String usuario;
+        int edad;
+        String ciudad;
+        String pais;
+        String correo;
+        String cadena = "";
+        System.out.println("Ingrese el nombre de usuario");
+        usuario = entrada.nextLine();
+        System.out.println("Ingrese su edad");
+        edad = entrada.nextInt();
+        entrada.nextLine();
+        System.out.println("Ingrese el nombre de su ciudad");
+        ciudad = entrada.nextLine();
+        System.out.println("Ingrese el nombre de su pais");
+        pais = entrada.nextLine();
+        System.out.println("Ingrese su correo");
+        correo = entrada.nextLine();
+        cadena = String.format("El nombre del usuario es: %s, tiene una edad de:"
+                + " %d años, vive en la ciudad de: %s, en país de: %s, su correo "
+                + "eletrónico es: %s", usuario, edad, ciudad, pais, correo); 
+        return cadena;
+    }
+    public static void crearTwitter(){
+        Scanner entrada = new Scanner(System.in);
+        String usuario;
+        String nombre;
+        String apellido;
+        int edad;
+        String ciudad;
+        String pais;
+        String idioma;
+        String correo;
+        System.out.println("Ingrese el nombre de usuario");
+        usuario = entrada.nextLine();
+        System.out.println("Ingrese sus nombres");
+        nombre = entrada.nextLine();
+        System.out.println("Ingrese sus apellidos");
+        apellido = entrada.nextLine();
+        System.out.println("Ingrese su edad");
+        edad = entrada.nextInt();
+        entrada.nextLine();
+        System.out.println("Ingrese el nombre de su ciudad");
+        ciudad = entrada.nextLine();
+        System.out.println("Ingrese el nombre de su pais");
+        pais = entrada.nextLine();
+        System.out.println("Ingrese su idioma");
+        idioma = entrada.nextLine();
+        System.out.println("Ingrese su correo");
+        correo = entrada.nextLine();
+        System.out.printf("El nombre del usuario es: %s, sus nombres son: %s, "
+                + "sus apellidos son: %s tiene una edad de: %d años, vive en la"
+                + " ciudad de: %s, en el país: %s, su idioma es: %s correo"
+                + " eletrónico es: %s\n",usuario, nombre, apellido, edad, ciudad,
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.